tests/read_mac/conftest.py

```diff
@@ -0,0 +1,44 @@
+"""
+    Pytest configuration used by the read MAC tests.
+"""
+
+def pytest_addoption(parser):
+    """
+        Adding arguments required for read MAC metadata testcase
+    """
+    parser.addoption(
+        "--iteration",
+        action="store",
+        type=int,
+        help="Number of image installing iterations",
+        required=True,
+    )
+
+    parser.addoption(
+        '--image1',
+        action='store',
+        type=str,
+        help='1st image to download and install',
+        required=True,
+    )
+
+    parser.addoption(
+        '--image2',
+        action='store',
+        type=str,
+        help='2nd image to download and install',
+        required=True,
+    )
+
+    parser.addoption(
+        '--minigraph1',
+        action='store',
+        type=str,
+        help='path to the minigraph1',
+    )
+    parser.addoption(
+        '--minigraph2',
+        action='store',
+        type=str,
+        help='path to the minigraph2',
+    )
```

Comment:
Can you put the options in a group like https://github.com/Azure/sonic-mgmt/blob/master/tests/route/conftest.py#L7.
The reason of this ask is because these options could conflict with options of other tests. We have hit this issue before. For example https://github.com/Azure/sonic-mgmt/pull/2385 was to address such conflicting issue.

Reply:
Done